winml/test/adapter/AdapterDmlEpTest.cpp

```diff
@@ -81,7 +81,7 @@ UniqueOrtSession CreateDmlSession() {
 
   THROW_IF_NOT_OK_MSG(
     winml_adapter_api->OrtSessionOptionsAppendExecutionProvider_DML(
-      session_options.get(), device.get(), queue.get(), false
+      session_options.get(), device.get(), queue.get(), false, true
```

Comment:
>, false, true [](http://example.com/codeflow?start=54&length=13)

Since we are here, can we add some comments per argument?

false, // metacommands
true, // bfc_allocator_enabled

Reply:
and in other places.